src/components/TxData.js

```diff
@@ -32,8 +40,17 @@ class TxData extends React.Component {
    * raw {boolean} if should show raw transaction
    * children {boolean} if should show children (default is hidden but user can show with a click)
    * tokens {Array} tokens contained in this transaction
+   * tokenClicked {Object} token clicked to be sent as props to the modal of unregistered token info
    */
-  state = { raw: false, children: false, tokens: [] };
+  state = {
+    raw: false,
+    children: false,
+    tokens: [],
+    tokenClicked: null,
+  };
+
+  // Array of token uid that was already found to show the symbol
+  tokensFound = [];
```

Comment:
Same comment as https://github.com/HathorNetwork/hathor-explorer/pull/4#discussion_r334092472

Reply:
They are both different cases because in the wallet we need to know if the token is unknown, so:

1. I was using it before because we weren't rendering authority inputs/outputs, so no need to add to the array a token that had only authorities (but it has changed now);
2. Here in the wallet I need to validate if each token is unknown and I use this `tokensFound` array not to execute this check if I have more than one input/output for the same token. 

So I think here in the wallet is necessary (I could have done this differently - e.g., using a set but this also works) but the explorer might have a refactor. What do you think?

Reply:
Ok, no problem.